verification/src/transaction_verifier.rs

```diff
@@ -693,9 +707,19 @@ impl<'a, DL: HeaderProvider> SinceVerifier<'a, DL> {
                     // parent of current block.
                     // pass_median_time(input_cell's block) starts with cell_block_number - 1,
                     // which is the parent of input_cell's block
-                    let cell_median_timestamp = self.parent_median_time(&info.block_hash);
-                    let current_median_time = self.block_median_time(&self.parent_hash);
-                    if current_median_time < cell_median_timestamp + timestamp {
+                    let proposal_window = self.consensus.tx_proposal_window();
+                    let parent_hash = self.tx_env.parent_hash();
+                    let epoch_number = self.tx_env.epoch_number(proposal_window);
+                    let hardfork_switch = self.consensus.hardfork_switch();
+                    let base_timestamp = if hardfork_switch
+                        .is_block_ts_as_relative_since_start_enabled(epoch_number)
+                    {
+                        self.parent_median_time(&info.block_hash)
+                    } else {
+                        self.parent_block_time(&info.block_hash)
+                    };
```

Comment:
Made a mistake after rebase.

```suggestion
                    let base_timestamp = if hardfork_switch
                        .is_block_ts_as_relative_since_start_enabled(epoch_number)
                    {
                        self.parent_block_time(&info.block_hash)
                    } else {
                        self.parent_median_time(&info.block_hash)
                    };
```